bin/crew

```diff
@@ -1256,9 +1209,9 @@ end
 
 def postinstall_command (args)
   args["<name>"].each do |name|
-    @pkgName = name
-    search @pkgName, true
-    if @device[:installed_packages].any? do |elem| elem[:name] == @pkgName end
+    if installed?(name)
+      @pkgName = name
```

Comment:
Do you even need to assign `@pkgName`?  You could just use name throughout.

Reply:
Change made